_pubs/animated-vega-lite.md

```diff
@@ -87,7 +93,7 @@ tags:
     </div>
     <div id="S1.p4" class="ltx_para">
       <p class="ltx_p">Animated Vega-Lite offers two abstractions of time that allow animations to compose with Vega-Lite’s existing grammars of static and interactive visualizations (<a href="#S4" title="4 A Grammar of Animation in Vega-Lite ‣ Animated Vega-Lite: Unifying Animation with a Grammar of Interactive Graphics" class="ltx_ref ltx_refmacro_autoref"><span class="ltx_text ltx_ref_tag">section&nbsp;4</span></a>).
-        From the perspective of interaction, time is an <emph class="ltx_text ltx_font_italic">event stream</emph>: a source of events analogous to <code class="ltx_verbatim ltx_font_typewriter">click</code>s and <code class="ltx_verbatim ltx_font_typewriter">keypress</code>es produced by a user.
+        From the perspective of interaction, time is an <emph class="ltx_text ltx_font_italic">event stream</emph>: a source of events analogous to <code class="ltx_verbatim ltx_font_typewriter">clicks</code> and <code class="ltx_verbatim ltx_font_typewriter">keypresses</code> produced by a user.
```

Comment:
this was technically not a typo because `click` and `keypress` are the names of the events